### PR TITLE
[Redis] Fix swagger issue discovered during C# code generation

### PR DESCRIPTION
**Pull request by SiddharthChatrolaMs**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/redis/resource-manager/Microsoft.Cache/2017-10-01/redis.json [hunk 1]
```diff
@@ -919,10 +919,10 @@
         "tags": [
           "Redis"
         ],
-        "operationId": "RedisLinkedServer_Create",
+        "operationId": "LinkedServer_Create",
```
Comment (lmazuel) [inline specification/redis/resource-manager/Microsoft.Cache/2017-10-01/redis.json]: Breaking for SDK. On purpose?

File: specification/redis/resource-manager/Microsoft.Cache/2017-10-01/redis.json [hunk 2]
```diff
@@ -1332,6 +1342,52 @@
         }
       ]
     },
+    "RedisCreateParameters": {
```
Comment (lmazuel) [inline specification/redis/resource-manager/Microsoft.Cache/2017-10-01/redis.json]: Your create does not have `tags` anymore. On purpose?

File: specification/redis/resource-manager/Microsoft.Cache/2017-10-01/redis.json (hunk 1 above)
Comment (lmazuel) [reply specification/redis/resource-manager/Microsoft.Cache/2017-10-01/redis.json]: I prefer the new naming, but it's still breaking. Should we consider this Swagger as a preview Swagger?

Comment (SiddharthChatrolaMs) [reply specification/redis/resource-manager/Microsoft.Cache/2017-10-01/redis.json]: .net SDK never had this API. This is new features so its not breaking change.

File: specification/redis/resource-manager/Microsoft.Cache/2017-10-01/redis.json (hunk 2 above)
Comment (SiddharthChatrolaMs) [reply specification/redis/resource-manager/Microsoft.Cache/2017-10-01/redis.json]: Nice catch. Fixed now.

File: specification/redis/resource-manager/Microsoft.Cache/2017-10-01/redis.json (hunk 1 above)
Comment (lmazuel) [reply specification/redis/resource-manager/Microsoft.Cache/2017-10-01/redis.json]: Note that C# is not the whole world :)
When a PR is merged to `current`, it means it's ready to be released for **all** SDK. Ruby, Node, Go, etc. This ApiVersion was merged 1 month ago, leaving room for SDK owner to use it.
FYI @salameer @marstr

Comment (lmazuel) [reply specification/redis/resource-manager/Microsoft.Cache/2017-10-01/redis.json]: So, I checked: Java, Ruby, Python, Node are not using this ApiVersion yet, so this is fine. You tell me C# is fine as well. Go ships it already, but they are planning a breaking release anyway, so they will sync this together.
So we are fine for this breaking.

Comment (SiddharthChatrolaMs) [reply specification/redis/resource-manager/Microsoft.Cache/2017-10-01/redis.json]: There is a requirement to send swagger pull request for API review, before we can actually roll out it in RP. We are still in middle of rolling this into RP. What is a way to inform other teams i.e. java, go etc to note use something till it is out in all regions?

**Review by lmazuel — APPROVED** [review]
Content LGTM. Breaking changes, but API version not shipped yet (I checked Java/C#/Ruby/Python/Node, and I discussed with Go team about this) and the breaking is a "good" breaking.

**Pull request merged**